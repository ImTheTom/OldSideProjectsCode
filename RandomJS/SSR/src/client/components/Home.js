import React from 'react';

const Home = () => {
	return (
    <div>
      <div>I'm the Best component</div>
	  <button onClick={()=>console.log("HEY")}>Press me!</button>
    </div>
  );
};

export default Home;