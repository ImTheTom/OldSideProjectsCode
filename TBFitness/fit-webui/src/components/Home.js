import React from "react";
import Header from "./Header";

class Home extends React.Component {
	render() {
		return (
			<div>
				<div>
					<Header />
				</div>
				<div>
					Home
				</div>
			</div>
		);
	}
}

export default Home;
