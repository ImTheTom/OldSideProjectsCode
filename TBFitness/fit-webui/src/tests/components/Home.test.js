import Home from "../../components/Home";
import React from "react";
import { shallow } from "enzyme";

test("should render create Home Page correctly", () => {
	const wrapper = shallow(<Home />);
	expect(wrapper).toMatchSnapshot();
});