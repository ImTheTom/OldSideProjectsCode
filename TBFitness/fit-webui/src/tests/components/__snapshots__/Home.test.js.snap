// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render create Home Page correctly 1`] = `
<div>
  <div>
    <Header />
  </div>
  <div>
    Home
  </div>
</div>
`;
